src/test/java/com/wikia/webdriver/testcases/notificationstests/messagewallnotifications/MessageWallNotificationsFollowersResponseTests.java

```diff
@@ -61,7 +62,7 @@ public void followerNotificationResponse_setup_2() {
     wall.writeTitle(title);
     wall.submit();
     wall.verifyMessageText(title, message, credentials.userName8);
-  }
+  }}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)